app/controllers/general_shopping_lists_controller.rb

```diff
@@ -0,0 +1,8 @@
+def index
+  @recipe_foods = RecipeFood.joins(:food)
+    .select('foods.name,
+                                     SUM(recipe_foods.quantity) as total_quantity,
+                                     SUM(recipe_foods.value) as total_value')
+    .group('foods.name')
+    .includes(:food)
+end
```

Comment:
- [ ] An error is thrown in an attempt to generate a shopping list, like below. Let's ensure to fix this:

![image](https://github.com/geekelo/domins-recipe_app/assets/12745474/32ab1ad8-1129-4c86-8095-dd04cecaf2ee)